src/main/java/org/mule/extension/aggregator/internal/storage/content/SimpleAggregatedContent.java

```diff
@@ -53,15 +53,24 @@ public void add(TypedValue newContent, Long timeStamp) {
 
   @Override
   public void add(TypedValue newContent, Long timeStamp, int sequenceNumber) {
-    sequencedElements.put(sequenceNumber, newContent);
+    if (sequencedElements.containsKey(sequenceNumber)) {
+      sequencedElements.get(sequenceNumber).add(newContent);
+    } else {
+      List<TypedValue> newList = new ArrayList<>();
+      newList.add(newContent);
+      sequencedElements.put(sequenceNumber, newList);
+    }
+
+    //sequencedElements.put(sequenceNumber, newContent);
```

Comment:
we usually do not leave comments in the code

Reply:
Ups. This was meant to be removed. Thanks